content/departures/amsterdam.md

```diff
@@ -11,12 +11,25 @@ weight: 2
 
 **Contact:** {{< cloakemail "amsterdam@kubetrain.io" >}}
 
-**Status:** {{< badge text="Finalizing" icon="compass" >}}
+**Status:** {{< badge text="Registration open" icon="star" >}}
 
-**Sponsor prospectus:** [link](/docs/KubeTrain_Sponsorship_prospectus_NL.pdf)
+**Tickets:** https://amsterdam.kubetrain.io
+
+### Journey
+
+📅 18 March, 2024  
+🚂 Eurostar 9364
 
-**Ticket sale starting in late January** 🎉
+```
+🚂 15:11 Asmterdam Centraal
+🚏 15:58 Rotterdam
+🚏 16:30 Antwerp
+🚉 17:08 Brussels
+🚉 18:40 Paris Gare du Nord
+```
+
+**Sponsor prospectus:** [link](/docs/KubeTrain_Sponsorship_prospectus_NL.pdf)
```

Comment:
We can probably remove this and the respective file. Let's not overcomplicate it